core/core.go

```diff
@@ -575,11 +614,13 @@ func cleanIfNeeded(bazelPath string) {
 
 // migrate will run Bazel with each flag separately and report which ones are failing.
 func migrate(bazelPath string, baseArgs []string, flags []string) {
+	var startupOptions = parseStartupOptions(baseArgs)
```

Comment:
Do we need this behavior for non-migrate code paths, too?

Reply:
This is only for running `bazel shutdown` and `bazel clean --expunge`, which is only used in migrate code path I think,